doc-surrealdb_versioned_docs/version-1.x/surrealql/parameters.mdx

```diff
@@ -30,103 +62,221 @@ let people = await surreal.query("SELECT * FROM article WHERE status INSIDE $sta
-                <code>$scope</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the name of the scope of a currently authenticated scope user.
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$session</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents values from the session functions as an object
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$before</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the value before a mutation on a field
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$after</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the value after a mutation on a field
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$value</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the value after a mutation on a field (identical to $after in the case of an event)
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$input</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the initially inputted value in a field definition, as the value clause could have modified the $value variable
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$this</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the current record in a subquery
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$parent</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the parent record in a subquery
-            </td>
-        </tr>
-         <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$event</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the type of table event triggered on an event.
-            </td>
-        </tr>
-    </tbody>
-</table>
\ No newline at end of file
+SurrealDB uses predefined variables. For that purpose, you can use those variables inside your query but you cannot declare new parameters using one of the following names:
```

Comment:
praise: Nice addition here. It really is easier to nagivate with the side menu.

Reply:
Ooh, I missed seeing the side menu. That is pretty nice.